src/main/java/com/redhat/devtools/lsp4ij/internal/telemetry/TelemetryManager.java

```diff
@@ -45,12 +47,12 @@ public TelemetryManager(Project project){
             Class.forName("com.redhat.devtools.intellij.telemetry.core.service.TelemetryService");
             telemetryService = new RedHatTelemetryService();
         } catch (Exception ignore) {
-            telemetryService = new NoOpTelemetryService();
+            telemetryService = new NoOpTelemetryService ();
         }
     }
 
-    public static TelemetryManager getInstance(@NotNull Project project) {
-        return project.getService(TelemetryManager.class);
+    public static TelemetryManager instance() {
+        return ApplicationManager.getApplication().getService(TelemetryManager.class);
```

Comment:
actually we need to keep it project scoped, else we'd send events for LS from other projects